desk.acc/sounds.s

```diff
@@ -1016,6 +1017,36 @@ loop5:  iny
         rts
 END_SOUND_PROC
 
+;;; ============================================================
+;;; Sound Routine: Bombdrop
+
+SOUND_PROC Bombdrop
+        ;; From ftp.apple.asimov.net/images/sound/ripped_off_routines.zip
+        ;; BRUN BOMBDROP(CALL3091)
+        ;; This routine is not relocatable & must be run from $0300(!)
+
+        LDA     #$00
+        STA     $FF
+        LDA     #$FF
+        STA     $FE
+loop1:  LDA     #$00
+        STA     SPKR
+        INC     SPKR
+        DEC     SPKR
+        LDY     #$05
+loop2:  LDX     $FF
+loop3:  DEX
+        BNE     loop3
+        DEY
+        BEQ     loop4
+        JMP     $0315      ;loop2
```

Comment:
This can use `loop2` - SOUND_PROC defines the assembly origin for the routine as $300 so hard-coding is not necessary.